Automattic/jetpack
# Boost: Update changelogs and readmes for 1.3.1-beta

Prepare for a 1.3.1 beta release by updating changelogs and readmes.

Also updates dependencies.

#### Changes proposed in this Pull Request:
* Update changelogs and readmes.

#### Jetpack product discussion
n/a

#### Does this pull request change what data or activity we track or use?
Yes; it adds the following data to our API requests when fetching Speed Scores, which are stored along with PageSpeed results:

- Jetpack Boost version.
- Which Jetpack Boost modules are enabled.

The purpose of this data is to give us a statistical overview of the effect each module (and version) has on speed scores.

#### Testing instructions:
* Check that the changelogs look ok.
* Check that the dependency changes are ok.
